test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app/DtaServiceAuthorizationClient.swift

```diff
@@ -0,0 +1,126 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import Foundation
+
+/// A struct to hold the settings used by DtaOidcAuthorizationClient
+struct DtaServiceAuthSettings {
+    var clientId: String
+    var clientSecret: String
+    var scope: String
+    var authority: String
+}
+
+typealias DtaServiceAuthorizationClientCallback = (Error?) -> ()
+
+class DtaServiceAuthorizationClient: NSObject, DtaAuthorizationClient {
+    /// Instance for `onAccessTokenChanged` property from the `AuthorizationClient` protocol.
+    public var onAccessTokenChanged: AccessTokenChangedCallback?
+    /// Instance for `errorDomain` property from the `DtaAuthorizationClient` protocol.
+    public let errorDomain = "com.bentley.display-test-app"
+    var issuerURL = URL(string: "https://ims.bentley.com")!
+    var authSettings: DtaServiceAuthSettings
+    let logger = PrintLogger()
+    var userInfo: NSDictionary?
+    var accessToken: String?
+    var expirationDate: Date?
+
+    init?(configData: JSON) {
+        let clientId = configData["IMJS_OIDC_CLIENT_ID"] as? String ?? ""
+        let clientSecret = configData["IMJS_OIDC_CLIENT_SECRET"] as? String ?? ""
+        let scope = configData["IMJS_OIDC_SCOPE"] as? String ?? "email openid profile organization itwinjs"
```

Comment:
@tcobbs-bentley, we should be moving away from ever using the `itwinjs` scope now that we have all of the APIs properly scoped and released with the ims authority.

In addition, the `email openid profile organization` should be all unnecessary with the proper set of scopes.

Reply:
@calebmshafer The default values here don't really work, so perhaps I should just delete them entirely. However, if what you say is true true, then it seems like our current iTwin Mobile SDK [instructions](https://github.com/iTwin/mobile-samples/blob/main/cross-platform/ClientID.md) for setting up an iTwin app are wrong. The `email openid profile organization` scopes are definitely present after following those instructions. Please let me know the proper way to register an app that will work on mobile, so that I can update our instructions.